website/src/main/tut/docs/users/installation.md

```diff
@@ -139,38 +139,75 @@ It is possible to use scalafix with scala-maven-plugin but it requires a custom
 
 ### Install semanticdb compiler plugin
 
-First, download the `semanticdb-scalac` compiler plugin which corresponds to your exact scala version of your project, down to the patch number.
+You can either install the semanticdb compiler plugin manually or let maven resolve the dependency for you.
 
-To begin with, it's recommended to install the coursier command line interface https://github.com/coursier/coursier.
+To install the plugin manually, first, download the `semanticdb-scalac` compiler plugin which corresponds to the exact scala version of your project(down to the patch number).
+
+It's recommended to do it via the [coursier command line interface](https://github.com/coursier/coursier) which simplifies downloading and launching library artifacts.
+
+To install coursier:
 
 ```
 wget https://github.com/coursier/coursier/raw/master/coursier && chmod +x coursier && ./coursier --help
 ```
 
-Coursier is a tool to download and launch library artifacts.
-Once you have coursier installed, assuming you are on {{ site.scala212 }}:
+Then, once you have coursier installed, assuming you are on {{ site.scala212 }}, download the plugin:
 
 ```
-coursier fetch --intransitive org.scalameta:semanticdb-scalac_{{ site.scala212 }}:{{ site.scalametaVersion }}
+coursier fetch --intransitive org.scalameta:semanticdb-scalac_{{ site.scala212 }}:2.1.7
```

Comment:
I'm a bit concerned of hardcoding this version since it'll be outdated once v0.6 is out in hopefully a couple weeks. I propose we keep this unchanged and add a hardcoded table instead to show which version of the compiler plugin matches which version of scalafix

```
| Scalafix | Scalameta | Scala          |
| -------- | --------- | -------------- |
| 0.5.10   | 2.1.7     | 2.12.4/2.11.12 |
| {{ site.version }} | {{ site.scalametaVersion }} | {{ site.scala212 }} / {{ site.scala211 }} | 
```

The last line might have wrong variable names, would be good to double check they are OK.

wdyt?